### PR TITLE
feat(docker): update to Amazonlinux

### PR DESCRIPTION
**Pull request by m0nhawk**
Jira Ticket: [VADC-752](https://ctds-planx.atlassian.net/browse/VADC-752)

### Improvements
migrating Docker image to Amazonlinux (needed for security compliance)

### Deployment changes

[VADC-752]: https://ctds-planx.atlassian.net/browse/VADC-752?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by EliseCastle23 — DISMISSED** [review]

File: server/server.go [hunk 1]
```diff
@@ -2,13 +2,9 @@ package server
 
 import (
 	"log"
-	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
```
Comment (pieterlukasse) [inline server/server.go:5]: is this change intentional? It is not mentioned in the PR description.

**Review by pieterlukasse — APPROVED** [review]
